lab_1_classify_by_unigrams/main.py

```diff
@@ -37,6 +61,16 @@ def calculate_mse(predicted: list, actual: list) -> float | None:
     :param actual: a list of actual values
     :return: the score
     """
+    if not isinstance(predicted, list) or not isinstance(actual, list) \
+            or len(predicted) != len(actual):
+        return None
+
+    mse = 0.0
+    for index, pred_value in enumerate(predicted):
+        mse += (pred_value - actual[index])**2
+
+    mse /= len(predicted)
```

Comment:
do not create the function

Reply:
What do you mean?

Reply:
I removed the line 72 and returned without reassigning mse